python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -413,6 +413,28 @@ def ElementwiseLeakyReluModule_basic(module, tu: TestUtils):
 # ==============================================================================
 
 
+class ElementwisePreluModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1, -1], torch.float32, True),
+        ([-1, -1], torch.float32, True),
+    ])
+    def forward(self, x, weight):
+        return torch.ops.aten.prelu(x, weight = torch.tensor([0.25]))
```

Comment:
What is the reason for passing the `torch.tensor` here instead of the `weight` argument of forward?

Reply:
sorry, was doing similar to lrelu.